pkg/security/scc/byrestrictions.go

```diff
@@ -31,20 +45,20 @@ func pointValue(constraint *securityapi.SecurityContextConstraints) int {
 	// strategies in order of least restrictive to most restrictive
 	switch constraint.SELinuxContext.Type {
 	case securityapi.SELinuxStrategyRunAsAny:
-		points += 4
+		points += runAsAnyUserWeight
```

Comment:
6 in one hand, half a dozen in another.  Seems like personal preference and requires an update for additions for either impl.  The map may keep it all organized right next to the constants.  Either way do we need a check (nil for map, default for switch) if something isn't recognized?  Probably a good idea.